src/main/java/com/aggregated/BuildAnnotatableCodePhase.java

```diff
@@ -649,7 +649,7 @@ private String buildConstructorBody(boolean isFromExisting) {
       superLine = new StringBuilder(IndentationUtils.get(IndentationUtils.INNER_BLOCK_TAB) + "super(");
       List<String> uniqueCleansed = removeDups(parentFields);
       for (int i = 0, n = uniqueCleansed.size(); i < n; i++) {
-        superLine.append(uniqueCleansed.get(i));
+        superLine.append(uniqueCleansed.get(i).trim());
```

Comment:
Cool, I think we can also use this method, too : [StringUtils::stripDoubleEndedNonAlphaNumeric](https://github.com/trgpnt/Java-Class-Annotatable-Constructor-Templater/blob/main/src/main/java/com/aggregated/StringUtils.java#L132)

Code approved for merging.